### DISCUSSION TITLE
7th Heaven doesnt start game when the game version is not English.

### DISCUSSION THREAD

**Discussion by Bl4ckh34d** [tsunamods-codes/7th-Heaven#217] [Q&A] [answered]
**Describe the bug**
Cannot start FF7 using 7th-Heaven. It seems the name ff7_en.exe is hard coded. In my steam the exe is called ff7_de.exe (German version). Changing the exe files name results in the warning that my game is cracked.

It appears you hardcoded ff7_en.exe as file name. But here is the thing: Spanish version is called "ff7_es.exe", French version is "ff7_fr.exe", German version is "ff7_de.exe", etc. At least add into a disclaimer that your piece of software only works for the ENGLISH version and nothing else or add check to your code for the other exe file names aswell. Changing the name of the exe file to what your software expects, results in an exception error concerning a "cracked" game. (I guess you do a checksum).

**To Reproduce**
Install FF7 from Steam for German language. Try starting the game via 7th Heaven. FAIL.

**Comment by Bl4ckh34d** [comment]
![Screenshot 2024-09-15 220150](https://github.com/user-attachments/assets/f069d65d-f9eb-4a19-a8d3-54f32b22d69d)

These are the hardcoded references I found.
I am not familiar with your code, so I can't give you a PR. I guess you will be faster in fixing this for all localizations.

**Comment by julianxhokaxhiu** [comment, marked as answer]
7th Heaven does and will always support the English edition of the game. There is no plan in supporting other languages at the current state.
